python/lsst/meas/algorithms/objectSizeStarSelector.py

```diff
@@ -317,18 +339,15 @@ def selectStars(self, exposure, sourceCat, matches=None):
         plotMagSize = lsstDebug.Info(__name__).plotMagSize             # display the magnitude-size relation
         dumpData = lsstDebug.Info(__name__).dumpData                   # dump data to pickle file?
 
-        # create a log for my application
-        logger = pexLogging.Log(pexLogging.getDefaultLog(), "meas.algorithms.objectSizeStarSelector")
-
         detector = exposure.getDetector()
         pixToTanXYTransform = None
         if detector is not None:
-            tanSys = detector.makeCameraSys(cameraGeom.TAN_PIXELS)
+            tanSys = detector.makeCameraSys(TAN_PIXELS)
             pixToTanXYTransform = detector.getTransformMap().get(tanSys)
```

Comment:
Is there any way we could share this distortion logic across multiple star selectors?  I suspect virtually all would want to make use of it.

Maybe an intermediate base class, or utility code somewhere else?

Could be a new issue if it's substantial work, but we should at least create the issue and describe it while these classes are fresh in your mind.